web/public/mgmt/config.yml

```diff
@@ -2166,6 +2166,7 @@ collections:
       preview: true
     fields:
       - { label: "Internal Notes", name: "notesMd", widget: "markdown", required: false }
+      - { label: "Id", name: "id", widget: "string", required: true, hint: "This is the id of the file." }
```

Comment:
Can we add the hint text to Task - All, and maybe update it to say, it should be the same as filename. Also, I think this should also use the write once widget. thoughts?

Reply:
i took it out of the task-all bc i dunno if that functionality of required would do anything to the current code, should i add it back?